README.md

```diff
@@ -1 +1,37 @@
-# hw01
+# Computing for the Social Sciences
+## The First Homework
+
+#### Who Am I? (... I am Jean Valjean!??)
+
+![Here I Am](Dhruv_Headshot.jpeg)
```

Comment:
Since the file "Dhruv_Headshot.jpeg" refers to a local path, other people would not be able to see the photo if they only download the readme.md file. I see that the rendering is fine because now the md file and jpg file are in the same folder in the repo. But the path, https://github.com/jessezorr/hw01/blob/master/Dhruv_Headshot.jpeg?raw=true is the url address for the uploaded photo file.